micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,65 @@
+--courserun migration from MicroMaster to MITxOnline
+INSERT INTO public.courses_courserun(
+    title,
+    courseware_id,
+    run_tag,
+    courseware_url_path,
+    start_date,
+    end_date,
+    enrollment_start,
+    enrollment_end,
+    expiration_date,
+    live,
+    created_on,
+    updated_on,
+    course_id)
+SELECT
+    mm_courserun.title,
+    mm_courserun.edx_course_key,
+    LEFT(CASE WHEN position('+' in mm_courserun.edx_course_key) > 0
+    THEN (regexp_match(mm_courserun.edx_course_key, '[^+]+$'))[1]
+    ELSE (regexp_match(mm_courserun.edx_course_key, '[^/]+$'))[1] END, 10) AS run_tag,
+    mm_courserun.enrollment_url,
+    mm_courserun.start_date,
+    mm_courserun.end_date,
+    mm_courserun.enrollment_start,
+    mm_courserun.enrollment_end,
+    mm_courserun.enrollment_end + INTERVAL '1 day',
+    NOT mm_courserun.is_discontinued,
+    coalesce(mm_courserun.enrollment_start, now()), --- need to confirm the default if enrollment_start is null
+    coalesce(mm_courserun.enrollment_end, now()),
+    pk_map.course_id
+FROM micromasters.courses_courserun AS mm_courserun
+JOIN public.micromasters_import_courseid AS pk_map ON mm_courserun.course_id = pk_map.micromasters_id
+LEFT JOIN public.courses_courserun AS mo_courserun ON mo_courserun.courseware_id = mm_courserun.edx_course_key
+WHERE mo_courserun.courseware_id IS NULL ---bring in course runs that don't exist in MITxOnline
+ON CONFLICT DO NOTHING; -- Do we want to update?
```

Comment:
If a courserun already exists in mitxonline, we don't want to update it with data from micromasters